### PR TITLE
Index and display parts

### LINKED ISSUE #128: Manuscript show page: Part of the manuscript section
Related to #9 and part of #125.

TBD: details related to this section of the manuscript show page:

vatican_graffle__public___show_page_for_mss_version_2

### Heading for each manuscript section
To match what's in the mockup above, the heading for each section (the identifying section text that is visible even when a section is collapsed) needs to be constructed from several elements and element attributes:

section-heading

Notes below correspond to the mockup above and use this manuscript as an example: 
`TEI` doc: https://digi.vatlib.it/tei/Barb.gr.221.xml
DigiVatLib catalog record: https://digi.vatlib.it/mss/detail/Barb.gr.221

Within the `<msDescription n="n">` element:

A: The text value of`<locus>47r-v. 49r-54v</locus>` - **row 46** in the spreadsheet

On a new line,

B: The text value of `<authorityAuthor date="129-199" rif="aut" type="person">Galenus, Claudius,</authorityAuthor>` - **row 47** of the spreadsheet (`rif="aut"` is the identifying attribute)

C: The value of the `date` attribute from the same element as in B above, followed by a period.

D: The value of the `<title scan="@1De diebus decretoriis (fragmenta)_supplied" type="supplied" value="De diebus decretoriis (fragmenta)">De diebus decretoriis (fragmenta)</title>` element where the type attribute is `supplied` - **row 53** of the spreadsheet. This value goes in italics (to help it stand out as a title).

### PR DESCRIPTION
**Pull request by cbeer**
screen shot 2018-06-27 at 16 08 11

Fixes #128 

Part of #137.

### CODE REVIEW

File: app/controllers/catalog_controller.rb [hunk 1]
```diff
@@ -4,8 +4,11 @@ class CatalogController < ApplicationController
   include Blacklight::Catalog
 
   configure_blacklight do |config|
-          config.show.oembed_field = :oembed_url_ssm
-          config.show.partials.insert(1, :oembed)
+  config.show.oembed_field = :oembed_url_ssm
```
Comment (mejackreed) [inline app/controllers/catalog_controller.rb]: Should this indention change have 2 additional spaces? Or is the GitHub diff messed up.

Comment (cbeer) [reply app/controllers/catalog_controller.rb]: I'm so confused about indentation in that file.

Comment (mejackreed) [reply app/controllers/catalog_controller.rb]: yeah.. #itsComplicated

File: config/locales/en.yml [hunk 2]
```diff
@@ -9,73 +9,76 @@ en:
     create:
       notice: Your Vatican IIIF resource has been successfully created.
       error: There was a problem with adding the Vatican IIIF resource.
+  catalog:
+    parts_default:
+      title: Parts of this manuscript
```
Comment (camillevilla) [inline config/locales/en.yml]: @cbeer Can we throw some Google Translate stub for this in it.yml as well?

Comment (cbeer) [reply config/locales/en.yml]: I figured we'd probably take all of our unknown strings and make the Vatican fill it in later?

**Pull request merged**